packages/expo-file-system/android/src/main/java/expo/modules/filesystem/next/FileSystemNextModule.kt

```diff
@@ -130,6 +130,10 @@ class FileSystemNextModule : Module() {
       Property("path") { directory ->
         directory.asString()
       }
+
+      Function("listAsRecords") { directory: FileSystemDirectory ->
```

Comment:
I don't know if I like this name or not. Let me think about a better one.

Reply:
Have you found a better one? 😄 If not, could you @aleqsio add a comment (on Android, iOS and TS) that this function is internal and will be removed in the future (when returning shared objects is supported).